seqr/views/apis/project_api_tests.py

```diff
@@ -166,3 +170,62 @@ def test_empty_project_page_data(self):
         self.assertDictEqual(response_json['genesById'], {})
         self.assertDictEqual(response_json['mmeSubmissionsByGuid'], {})
         self.assertDictEqual(response_json['locusListsByGuid'], {})
+
+
+# Tests for AnVIL access disabled
+class LocalProjectAPITest(AuthenticationTestCase, TestHelpers.ProjectAPITest):
+    fixtures = ['users', '1kg_project', 'reference_data']
+    PROJECT_COLLABORATORS = [{'dateJoined': '2017-03-12T23:09:54.180Z', 'displayName': 'Test Manager User', 'email': 'test_user_manager@test.com', 'firstName': 'Test Manager User', 'hasEditPermissions': True, 'hasViewPermissions': True, 'id': 11, 'isAnvil': False, 'isStaff': False, 'lastName': '', 'username': 'test_user_manager'},
+{'dateJoined': '2017-03-12T23:09:54.180Z', 'displayName': 'Test Non Staff User', 'email': 'test_user_no_staff@test.com', 'firstName': 'Test Non Staff User', 'hasEditPermissions': False, 'hasViewPermissions': True, 'id': 12, 'isAnvil': False, 'isStaff': False, 'lastName': '', 'username': 'test_user_non_staff'}]
+
+
+ANVIL_PROJECT_COLLABORATORS = [{'dateJoined': '2017-03-12T23:09:54.180Z', 'displayName': 'Test Manager User', 'email': 'test_user_manager@test.com', 'firstName': 'Test Manager User', 'hasEditPermissions': False, 'hasViewPermissions': True, 'id': 11, 'isAnvil': True, 'isStaff': False, 'lastName': '', 'username': 'test_user_manager'},
```

Comment:
nitpicky, but its more readable if you define constants at the top of the file

Reply:
Okay. Then, I will also define one constant for local collaborators so that two constants sit side-by-side.

Reply:
Since the three scenarios will have different collaborator lists, I delete the constant and use the literal data.